tests/test_core_multimap.py

```diff
@@ -1,7 +1,6 @@
 from typing import Sequence, NamedTuple
```

Comment:
Change this test back for the original Multimap, we had a new copy for the new IonPyDict.